requirements-devel.txt

```diff
@@ -59,7 +59,7 @@ krb5==0.2.0
     # via pyspnego
 mccabe==0.6.1
     # via flake8
-osbs-client @ git+https://github.com/containerbuildsystem/osbs-client@ff6880d7620aa6837c856528f8b97aa4b123c490
+osbs-client @ git+https://github.com/hjmodi/osbs-client@04e80a4b0fd395f442a82e52f2fff5da0bdac9a7
```

Comment:
:eyes: just watching this to avoid merge from a custom fork

Reply:
updated reference to the upstream repo